routers/tournamentStatusRouter.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+const express = require('express')
+
+const mhubConnection = require('../Utils/mhubConnection')
+const getMatchLogic = require('../logic/getMatchLogic')
+
+const { authroizationMiddlware } = require('@first-lego-league/ms-auth')
+
+const adminAction = authroizationMiddlware(['admin', 'development'])
+
+exports.getRouter = function () {
+  const router = express.Router()
+
+  router.get('/current', (req, res) => {
+    getMatchLogic.getCurrentMatch().then(data => {
+      if (data) {
+        mhubConnection.publishUpdateMsg('CurrentMatches', data)
+        res.send(data)
+      } else {
+        res.sendStatus(404)
+      }
+    })
+  })
+
+  router.get('/upcoming', (req, res) => {
+    Promise.all(getMatchLogic.getNextMatches()).then(data => data.filter(data => data)).then(data => {
+      if (data && data.length > 0) {
+        mhubConnection.publishUpdateMsg('UpcomingMatches', data)
+        res.send(data)
+      } else {
+        res.sendStatus(404)
+      }
+    })
+  })
+
+  router.get('/matchNumber', (req, res) => {
+    res.send(getMatchLogic.getMatch().toString())
```

Comment:
You can just use sendJson

Reply:
Also with res.json it needs to be a string so there is no difference.

Reply:
No, it is not.
[see here](https://expressjs.com/en/4x/api.html#res.json)

Reply:
No, its not. [see here](https://expressjs.com/en/4x/api.html#res.json)